lib/Fencer/Rules.hs

```diff
@@ -21,18 +27,36 @@ import qualified Data.Yaml as Yaml
 
 import Fencer.Types
 
+data LoadRulesError
+  = ParseException FilePath Yaml.ParseException
+  | IOEx IOException
+
+instance Show LoadRulesError where
```

Comment:
Where is this instance used?

As a rule, `Show` instances are used for debugging and should be autoderived. If you want to do something pretty, you can put it into `showErrors`.

Reply:
There is no instance of `Show` for `Yaml.ParseException`, hence the manually written instance.

The instance is needed to print out error messages on an error.